src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -204,13 +215,15 @@ export class CreateBaselineModal extends Component {
     }
 
     renderActions() {
-        const { selectedBaselineIds, entities } = this.props;
+        const { selectedBaselineIds, selectedHSPIds, entities } = this.props;
         const { baselineName, copyBaselineChecked, copySystemChecked } = this.state;
         let actions;
 
         if (baselineName === ''
             || (copyBaselineChecked && selectedBaselineIds.length === 0)
-            || (copySystemChecked && entities && entities.selectedSystemIds !== undefined && entities.selectedSystemIds.length === 0)
+            || (copySystemChecked &&
+                ((entities && entities.selectedSystemIds !== undefined && entities.selectedSystemIds.length === 0) && (selectedHSPIds.length === 0))
```

Comment:
You could replace this logic with Lodash's `isEmpty` to simplify it -- https://lodash.com/docs/4.17.15#isEmpty